opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -364,21 +371,29 @@ namespace Opm {
       using vec_size_t = std::vector<std::size_t>;
       EclipseGridLGR() = default;
       EclipseGridLGR(const std::string& self_label, const std::string& father_label_, 
-                     int father_lgr_level, size_t nx, size_t ny, size_t nz,
-                     const vec_size_t& father_lgr_index);
+                     size_t nx, size_t ny, size_t nz, 
+                     vec_size_t father_lgr_index, const std::array<int,3>& low_fahterIJK_, 
```

Comment:
You're still passing `father_lgr_index` by value.  Is that intentional?  If so, please add a comment for why that is.

Reply:
no. I have fixed it.